run_configs/lfric/atm.py

```diff
@@ -247,6 +247,7 @@ def file_filtering(config):
             kernel_roots=[state.build_output / 'lfric' / 'kernel'],
             transformation_script=get_transformation_script,
             cli_args=[],
+            api="dynamo0.3",
```

Comment:
New to this repo, so don't know what the style guidelines are. But this new argument is repeated in multiple calls to `psyclone`. Is it better to add it as a constant somewhere to keep things DRY?

```
DEFAULT_PSYCLONE_API = "dynamo0.3"
```

?

Reply:
Also, I was told that the "dynamo0.3" API name is deprecated as of PSyclone 2.5. It is replaced by "lfric".

Reply:
Anyway, that was a small change, so I've added an API to lfric_common.py.